### PR TITLE
Support delegateInfo RPC

### PR DESCRIPTION
**Pull request by camfairchild**
This PR implements the delegate info rpc to get info on delegates faster.  

https://github.com/opentensor/paratensor/pull/36

### CODE REVIEW

**Review by unconst — COMMENTED** [review]
LGTM

File: bittensor/_cli/commands/delegates.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def show_delegates( delegates: List['bittensor.DelegateInfo'], width: Optional[i
             str(delegate.hotkey_ss58),
             str(delegate.take),
             str(delegate.owner_ss58),
-            str(delegate.nominators),
+            str(len(delegate.nominators)),
```
Comment (Eugene-hu) [inline bittensor/_cli/commands/delegates.py:41]: what does `delegate.nominators` contain? a list of hotkeys that have also nominated the same delegator?

**Review by Eugene-hu — APPROVED** [review]
LGTM

File: bittensor/_cli/commands/delegates.py (hunk 1 above)
Comment (camfairchild) [reply bittensor/_cli/commands/delegates.py:41]: It's a list of tuples of `(coldkey_ss58, stake delegated)` for the delegate hotkey

Comment (camfairchild) [reply bittensor/_cli/commands/delegates.py:41]: Essentially they "nominate" the delegate

**Pull request merged**